distributions/aws/examples/storage/efs/auto-efs-setup.py

```diff
@@ -553,7 +562,7 @@ def footer():
     help='Your cluster name (eg: mycluster-1)',
     required=True
 )
-EFS_FILE_SYSTEM_NAME_DEFAULT = "KubeflowEfs"
+EFS_FILE_SYSTEM_NAME_DEFAULT = rand_name("Kubeflow-efs")
```

Comment:
Are we not deleting these EFS volume ?

Reply:
Not in this script no, so if you run this script multiple times it reuses the same volume. We need to randomize this name to be able to use these in the test automation.

Reply:
More details are in the comment below by Alexandre.